oneanddone/tests/functional/tests/test_task_assignment.py

```diff
@@ -0,0 +1,57 @@
+# This Source Code Form is subjectfrom django import forms to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.home import HomePage
+
+
+class TestAvailableTasks():
+    @pytest.mark.nondestructive
+    def test_create_tasks(self, base_url, selenium, task, random_title):
+
+        admin_user = {'email': "varna.suresh@outlook.com",
+                      'password': "varna123",
+                      'name': "Varna",
+                      'username': "Varna",
+                      'url': 'http://www.mozilla.org/'
+        }
+
+        home_page = HomePage(selenium, base_url).open()
+        assert not home_page.is_user_logged_in
+        edit_profile = home_page.login(admin_user)
+        assert edit_profile.is_user_logged_in
+        home_page.click_available_tasks()
+        home_page.click_create_task(random_title)
+
+    @pytest.mark.nondestructive
+    def test_assign_tasks(self, base_url, selenium, task, new_user, random_title):
```

Comment:
This test depends on `test_create_tasks` succeeding. We can't guarantee this test will even run first. It's good practice for tests to take care of all data they rely on, and can run in isolation. This should be a single test that logs in, creates a task, assigns it to one user, and then checks that another user cannot take it. The creation and assignment of the task can be taken care of in a similar way to https://github.com/mozilla/oneanddone/blob/master/oneanddone/tests/functional/conftest.py#L69-L73